sdk-setup/src/mb2

```diff
@@ -2686,9 +2686,10 @@ END
 init_patch_wrapper() (
     local wrapper=$1
     local patchset_id=$2
-
+    HOST_GIT=$(which git)
```

Comment:
```suggestion
    local host_hit=$(which git)
```

Reply:
now if we use `local host_git` here (I assume "hit" was a typo), we need to change git() to use `"$host_git"?` (lower case), which means we need to use lower case also in the wrapper. Are you ok with that?

Reply:
hm yeah, the use in the initialization again... so I would return to my original suggestion and use `command git` in the initialization code to avoid the issue you hit, i.e., to make it use the real git instead of the wrapper function in that context.